README.md

```diff
@@ -788,31 +788,31 @@ The ethtool configuration supports the following options:
     coalesce:
       adaptive_rx: true|false  # optional
       adaptive_tx: true|false  # optional
-      pkt_rate_high: 0  # optional mininum=0 maximum=0xffffffff
-      pkt_rate_low: 0  # optional mininum=0 maximum=0xffffffff
-      rx_frames: 0  # optional mininum=0 maximum=0xffffffff
-      rx_frames_high: 0  # optional mininum=0 maximum=0xffffffff
-      rx_frames_irq: 0  # optional mininum=0 maximum=0xffffffff
-      rx_frames_low: 0  # optional mininum=0 maximum=0xffffffff
-      rx_usecs: 0  # optional mininum=0 maximum=0xffffffff
-      rx_usecs_high: 0  # optional mininum=0 maximum=0xffffffff
-      rx_usecs_irq: 0  # optional mininum=0 maximum=0xffffffff
-      rx_usecs_low: 0  # optional mininum=0 maximum=0xffffffff
-      sample_interval: 0  # optional mininum=0 maximum=0xffffffff
-      stats_block_usecs: 0  # optional mininum=0 maximum=0xffffffff
-      tx_frames: 0  # optional mininum=0 maximum=0xffffffff
-      tx_frames_high: 0  # optional mininum=0 maximum=0xffffffff
-      tx_frames_irq: 0  # optional mininum=0 maximum=0xffffffff
-      tx_frames_low: 0  # optional mininum=0 maximum=0xffffffff
-      tx_usecs: 0  # optional mininum=0 maximum=0xffffffff
-      tx_usecs_high: 0  # optional mininum=0 maximum=0xffffffff
-      tx_usecs_irq: 0  # optional mininum=0 maximum=0xffffffff
-      tx_usecs_low: 0  # optional mininum=0 maximum=0xffffffff
+      pkt_rate_high: 0  # optional minimum=0 maximum=0xffffffff
```

Comment:
@liangwen12year there are many mininum>minimum edits. Are these all typos or `mininum` was written intentionally?

Reply:
mininum>minimum? What do you mean exactly ?

Reply:
I think that `mininum` is a typo, thanks fo correcting it.